contrib/intel/jenkins/Jenkinsfile

```diff
@@ -122,6 +122,23 @@ pipeline {
                      }
 
                  }
+
+                 stage('ze-shm') {
+                     agent {node {label 'eth'}}
+                     steps{
+                         withEnv(['PATH+EXTRA=/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin/:$PYTHONPATH'])
+                         {
+                           sh """
+                             env
+                             (
+                                 cd ${env.WORKSPACE}/contrib/intel/jenkins/
+                                 python3.7 runtests.py --prov=shm --ofi_build_mode='dbg'
```

Comment:
Once  it is verified that this test is working, we can remove the similar call from the eth-udp-shm stage.